Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,42 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol/archive/v2.2.0.tar.gz"
+  sha1 "fc61dfe6993cf4aa57410d98faabb754a64065d3"
```

Comment:
You'll need to bump any checksums to ` sha256 ` at this point. We merged stricter rules on checksums for new formulae a couple of weeks back.